### PR TITLE
fix: Fixing clear event listener issue

### PR DESCRIPTION
**Pull request by Abhishk123**

### CODE REVIEW

File: cypress/fixtures/fireboltMocks/clearEventData.json [hunk 1]
```diff
@@ -42,5 +42,9 @@
     "SECONDSCREEN_ONCLOSEREQUEST_EVENT":{
         "event": "secondscreen.onCloseRequest",
         "firstParty": false
+    },
+    "ADVERTISING_ONSKIPRESTRICTIONCHANGED_EVENT":{
+        "event": "manage_advertising.onSkipRestrictionChanged",
+        "firstParty": false
     }
```
Comment (neeradanelxsi) [inline cypress/fixtures/fireboltMocks/clearEventData.json:49]: why is firstParty false ? this looks to be a manage event . 

is this for clearing event listener on FCA , but for a manage event ?

Comment (neeradanelxsi) [reply cypress/fixtures/fireboltMocks/clearEventData.json:49]: if this is applicable only for xumo config module, cant we place this in xumo module and not directly in fcs >?

**Review by jbigel — APPROVED** [review]

File: cypress/fixtures/fireboltMocks/clearEventData.json (hunk 1 above)
Comment (Abhishk123) [reply cypress/fixtures/fireboltMocks/clearEventData.json]: move this to config module and firstParty is false because making a call to 3rd party app

**Review by neeradanelxsi — APPROVED** [review]

**Pull request merged**